.github/workflows/build.yml

```diff
@@ -11,37 +11,16 @@ on:
       - '*'
 
 jobs:
-  build_novacustom_ec:
```

Comment:
@mkopec I think we can still build EC regardless if EC sync is used or not. The "Obtain EC" step is removed and the dependency too, so it should be fine.